idx/memory/memory.go

```diff
@@ -407,13 +431,12 @@ func (m *MemoryIdx) Load(defs []schema.MetricDefinition) int {
 	return num
 }
 
-func (m *MemoryIdx) add(def *schema.MetricDefinition) idx.Archive {
+func (m *UnpartitionedMemoryIdx) add(def *schema.MetricDefinition) idx.Archive {
 	path := def.NameWithTags()
-
 	schemaId, _ := mdata.MatchSchema(path, def.Interval)
 	aggId, _ := mdata.MatchAgg(path)
 	irId, _ := IndexRules.Match(path)
-	sort.Strings(def.Tags)
```

Comment:
why are we removing this? this looks like a change that requires special attention, and not slip in via a commit named "add new PartitionedMemoryIdx".
ideally we don't make changes in this PR that are unrelated to the partitioning.

Reply:
`def.NameWithTags()` already sorts the tags.